core/storage/exploration/gae_models.py

```diff
@@ -239,6 +239,61 @@ def has_reference_to_user_id(cls, unused_user_id):
         return False
 
 
+class ExplorationMathRichTextInfoModel(base_models.BaseModel):
+    """Temporary Storage model for storing information useful while generating
+    images for math rich-text components in explorations.
```

Comment:
Explain when this will be deleted. Create a TODO and link to an issue.

Reply:
done